rustv1/examples/s3/src/bin/if-modified-since.rs

```diff
@@ -15,107 +15,168 @@ use tracing::{error, warn};
 const KEY: &str = "key";
 const BODY: &str = "Hello, world!";
 
-/// Lists your tables in DynamoDB local.
+/// Demonstrate how `if-modified-since` reports that matching objects haven't
+/// changed.
+///
+/// *Steps*
```

Comment:
(nit) these comments are markdown, so prefer our markdown style - # or ## if its a heading, or \**...\** for bold (\_...\_ for italic), and \- for unordered lists (1. for ordered)

Reply:
Thanks, fixing now.